README.md

```diff
@@ -1,15 +1,88 @@
-# torch-FastText: Efficient text classification with PyTorch
+## torchFastText
```

Comment:
Why change the title ?

Reply:
no specific reason ; just feels better to me without "-" (very subjective).

Reply:
I was just talking about the text, I think we should keep the level (and the mini description)  : 

```suggestion
# torchFastText : Efficient text classification with PyTorch
```